src/components/CheckForm/checkForm.hooks.ts

```diff
@@ -95,3 +102,70 @@ export function useCheckForm({ check, checkType, onTestSuccess }: UseCheckFormPr
     handleInvalid,
   };
 }
+
+function findFieldToFocus(errs: FieldErrors<CheckFormValues>) {
```

Comment:
This file is called `checkForm.hooks.ts` but these new functions are not hooks, maybe something to consider when we work on [this](https://github.com/grafana/synthetic-monitoring-app/issues/848) ticket

Reply:
Good point! I've separated them out to `checkForm.utils.ts`.